provision.sh

```diff
@@ -58,9 +58,9 @@ sudo systemctl restart NetworkManager
 
 echo "Installing oc client"
 cd $HOME
-curl -OL https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
-tar -zxf oc.tar.gz
-rm -fr oc.tar.gz
+curl -OL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz
+tar -zxf openshift-client-linux.tar.gz
+rm -fr openshift-client-linux.tar.gz
 sudo mv $HOME/oc /usr/local/bin
```

Comment:
Did not see this at first, but can't this be just `curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar zx` ?
ACK either way.